### DISCUSSION TITLE
Div Breakpoints instead of Screen Breakpoints

### DISCUSSION THREAD

**Discussion by timheerwagen** [tailwindlabs/tailwindcss#7893] [Help]
Is it possible to use taildwind breakpoints that refer to the width of a div and not to the width of the screen?

For example screen width is 1024px (lg), but div is only 640px (sm), so take breakpoint sm for styling inside the div.

**Comment by browner12** [comment]
not yet.

https://caniuse.com/css-container-queries

but someday
